packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -2094,6 +2094,69 @@ function validateTailOptions(
   }
 }
 
+function validateSuspenseListNestedChild(childSlot: mixed) {
+  if (__DEV__) {
+    let isArray = Array.isArray(childSlot);
+    let isIterable = !isArray && typeof getIteratorFn(childSlot) === 'function';
+    if (isArray || isIterable) {
+      warning(
+        false,
+        'A nested %s was passed to <SuspenseList />. This whole array ' +
+          'will be treated as a single row in the list. If this is what you ' +
+          'want, wrap it in a fragment e.g. <>{array}</>. If you meant to ' +
+          'merge multiple arrays to a single list, try using React.Children.toArray(). ' +
```

Comment:
Maybe `toArray(array)`. It took me a bit to realize what you’re suggesting to do here.

Reply:
It would be more like `toArray([firstArray, secondArray])` but it's kind of hard to give a precise recommendation without knowing how this happened.